brianlong/www.validators.app
# 178709228 pipeline for asn

#### What's this PR do?
create ASN pipeline to gather stats

#### How should this be manually tested?
run script/append_ip_geo_data.rb to make sure you have some Ips
run dev/asn_logic_demo.rb and see if there's any AsnStats in the db.
Check if all fields are populated
run tests

#### What are the relevant tickets?
https://www.pivotaltracker.com/story/show/178709228

#### Task completed checklist:
- [Y] Is there appropriate test coverage?
- [Y] Did I take into consideration possible security vulnerabilities?
- [Y] Are the controllers secure?
- [Y] Was every important matter documented?

#### Things to watch out for
- [Y] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
